Utilities/Maintenance/AuthorsChangesSince.py

```diff
@@ -186,25 +186,25 @@ def formatted_current_author():
             prefix = line.split(":")[0].strip()
             commit = line.split(":")[-1]
             if prefix == "BUG":
-                description = line.split(":")[1]
+                description = line.partition(":")[2].rpartition(":")[0].strip()
```

Comment:
Does Python support writing helper functions? If so, I guess this may be a candidate! `make_description (const string & line)` 😸

Reply:
Can you possibly tell me what a line of the git log looks like? There's always a colon at the beginning and a colon at the end, right? Do you have an example?

Reply:
> There is not a significant improvement by creating a function.

Well, personally I do think a `make_description` function could in this case improve readability and maintainability, as I see the same `description =` line of code repeated six times now. But I leave it up to you if you think otherwise.

Reply:
Maybe defining a function elsewhere could make sense when it is used in various places, but I do not think the additional indirection and complexity significantly improves readability and readability in this case.